src/Mapper/Mapper.test.ts

```diff
@@ -0,0 +1,360 @@
+// tslint:disable: max-classes-per-file no-unused-expression
+import { Observable } from "rxjs/Observable";
+import "rxjs/add/observable/of";
+import { spy, stub } from "sinon";
+
+import faker from "faker";
```

Comment:
Not really something to fix but if you want to streamline your faker calls a bit, you can also do `import { lorem, random } from "faker"` instead of importing everything from the library  (you'd have to change the other tests in that file that use Faker as well)

Reply:
Good call! Just submitted a change for this